extensions/ql-vscode/src/databases/ui/db-panel.ts

```diff
@@ -363,6 +369,15 @@ export class DbPanel extends DisposableObject {
     await this.dbManager.renameList(dbItem, newName);
   }
 
+  private async removeItem(treeViewItem: DbTreeViewItem): Promise<void> {
+    if (treeViewItem.dbItem === undefined) {
+      throw new Error(
+        "Not a removable database item. Please select a valid item.",
+      );
+    }
+    await this.dbManager.removeDbItem(treeViewItem.dbItem);
+  }
```

Comment:
Shall we also clean expanded items like we said? `updateDbItemExpandedState` now does that.